### PR TITLE
GreenSock TweenMax fix RoughEase

### PR DESCRIPTION
**Pull request by lucap86**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://greensock.com/docs/#/HTML5/GSAP/Easing/RoughEase/config/>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: greensock/index.d.ts [hunk 1]
```diff
@@ -298,7 +298,7 @@ declare type RoughEaseConfig = {
 declare class RoughEase extends Ease {
     public static ease: RoughEase;
     constructor(vars: RoughEaseConfig);
-    public config(steps: number): SteppedEase;
+    public config(steps: RoughEaseConfig): RoughEase;
```
Comment (DanielRosenwasser) [inline greensock/index.d.ts]: And also, can you make it optional?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: greensock/index.d.ts (hunk 1 above)
Comment (lucap86) [inline greensock/index.d.ts]: make it optional should be enough..
isnt'it?

**Comment by lucap86** [comment]
@DanielRosenwasser can you please unlock pull request ?
as I said before, make it optional should be enough
@ProbablePrime has done the job in  commit #8826814f37249f56ccd08d19d578ed20b4bf2a24

**Comment by bowdenk7** [comment]
@DanielRosenwasser not sure what's going on here. Can you resolve?

**Comment by ProbablePrime** [comment]
Personally i'd say close this and defer to my branch. Otherwise you'd have to merge this then mine.

**Comment by aozgaa** [comment]
Let's wait on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14332, and if that fails then take this smaller change.

**Comment by minestarks** [comment]
Closing since it looks like the work is already done in commit 8826814f37249f56ccd08d19d578ed20b4bf2a24

**Comment by ste-vg** [comment]
commit 8826814 didn't include the `RoughEaseConfig` change to the steps type.

**Pull request closed without merging**